.github/workflows/CompatHelper.yml

```
@@ -0,0 +1,16 @@
+name: CompatHelper
+on:
+  schedule:
+    - cron: '00 00 * * *'  # Everyday at midnight
```

I don't think you need `00` here?

```suggestion
    - cron: '0 0 * * *'  # Everyday at midnight
```

---

yeah I think you're right. I just copied this file from JLSO I think